source/_integrations/vizio.markdown

```diff
@@ -1,12 +1,13 @@
 ---
-title: Vizio SmartCast TV
+title: Vizio SmartCast
 description: Instructions on how to integrate Vizio SmartCast TVs and sound bars into Home Assistant.
 logo: vizio-smartcast.png
 ha_category:
   - Media Player
 ha_release: 0.49
 ha_iot_class: Local Polling
 ha_config_flow: true
+ha_quality_scale: platinum
 ha_codeowners:
   - '@raman325'
 ---
```

Comment:
Update `required` to `true` for `device_class`, see https://github.com/home-assistant/home-assistant/pull/31975

Reply:
That update was a mistake in the parent PR, will fix

Reply:
Actually nevermind, it was not a mistake. The change was made because when a user uses config flow through the UI, that field is required to be filled out. In config import, that field is optional because a default is provided. So the documentation is correct.

Reply:
Okay thanks for the clarification.

Reply:
Alternatively, let me know if you would like me to revert the field to Optional in the integration code. The Optional vs Required decision has no impact to the UI config flow today because vol.In resolves to a drop down choice list in the UI and at least one value has to be selected. I switched it to required as a safety precaution, but it may not be necessary.

Reply:
Well I think it is best to revert it back to optional since it does not impact on the UI and code wise it does not make a difference either. Required without default value is basically the same as optional with a default value, both are accessed via `dict[key]` and the respective value needs to be set for a successful setup.

Reply:
Sounds good!